commands/hello.md

```diff
@@ -1,20 +1,21 @@
-Switch to a different protocol, optionally authenticating and setting the connection's name, or provide a contextual client report.
+Switch to a different protocol, optionally authenticating and setting the
+connection's name, or provide a contextual client report.
 
-Redis version 6 and greater supports two protocols: the old protocol,
-RESP2, and a new one introduced with Redis 6, RESP3. RESP3 has certain
-advantages since when the connection is in this mode, Redis is able to reply
-with more semantical replies: for instance, `HGETALL` will return a *map type*,
-so a client library implementation no longer requires to know in advance to
-translate the array into a hash before returning it to the caller. For a full
-coverage of RESP3, please
+Redis version 6 and above supports two protocols: the old protocol, RESP2, and
+a new one introduced with Redis 6, RESP3. RESP3 has certain advantages since
+when the connection is in this mode, Redis is able to reply with more semantical
+replies: for instance, `HGETALL` will return a *map type*, so a client library
+implementation no longer requires to know in advance to translate the array into
+a hash before returning it to the caller. For a full coverage of RESP3, please
 [check this repository](https://github.com/antirez/resp3).
 
 In Redis 6 connections start in RESP2 mode, so clients implementing RESP2 do
-not need to updated or changed (nor there are short term plans to drop support for
-RESP2, but future version may default to RESP3).
+not need to updated or changed. There are short term plans to drop support for
+RESP2, although future version may default to RESP3.
```

Comment:
did you notice you negated the sentence?
```suggestion
not need to updated or changed. There are no short term plans to drop support for
RESP2, although future version may default to RESP3.
```

Reply:
No, I haven't (noticed).